examples/asset-upload/.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 module.exports = {
-    extends: ['@frontify/eslint-config-react'],
+    extends: ['@frontify/eslint-config-react', 'plugin:jsx-a11y/recommended'],
```

Comment:
```suggestion
    extends: ['@frontify/eslint-config-react', 'plugin:jsx-a11y/strict'],
```
?

Reply:
Will have a look 👍

Reply:
They want you to think it is enabled 😄 

<img width="451" alt="Screenshot 2023-09-08 at 15 48 58" src="https://github.com/Frontify/guideline-blocks/assets/6846950/29fc8624-5b9a-494f-b136-e7b1d2a71e26">